storm-core/src/jvm/org/apache/storm/utils/Time.java

```diff
@@ -98,10 +104,25 @@ public static long currentTimeMillis() {
             return System.currentTimeMillis();
         }
     }
-    
+
+    public static long toMillis (int secs) {
+        return 1000*(long) secs;
+    }
+    public static long toMillis (String secs) {
+        return 1000*Long.parseLong(secs);
+    }
+
     public static int currentTimeSecs() {
         return (int) (currentTimeMillis() / 1000);
     }
+
+    public static int delta(int timeInSeconds) {
+        return Time.currentTimeSecs() - timeInSeconds;
+    }
+
+    public static long deltaMs(long timeInMilliseconds) {
```

Comment:
Most other places in this file uses Millis not Ms, or milliseconds is considered the default.  Could we follow and rename this deltaMillis, or rename all of the others to use Ms instead of Millis?